sequencer/src/bin/cdn-broker.rs

```diff
@@ -0,0 +1,99 @@
+//! The following is the main `Broker` binary, which just instantiates and runs
+//! a `Broker` object.
+
+use anyhow::{Context, Result};
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use cdn_broker::{Broker, Config, ConfigBuilder};
+use clap::Parser;
+
+use hotshot::traits::implementations::{KeyPair, ProductionDef, WrappedSignatureKey};
+use hotshot_types::traits::node_implementation::NodeType;
+use hotshot_types::traits::signature_key::SignatureKey;
+use sequencer::SeqTypes;
+use sha2::Digest;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+/// The main component of the push CDN.
+struct Args {
+    /// The discovery client endpoint (including scheme) to connect to.
+    /// With the local discovery feature, this is a file path.
+    /// With the remote (redis) discovery feature, this is a redis URL (e.g. `redis://127.0.0.1:6789`).
+    #[arg(short, long)]
```

Comment:
Shouldn't we add env vars to enable configuration of the new binaries?

Reply:
Good idea, yes, will do